integration_test/third_party_apps_data/applications/vault/metadata.yaml

```diff
@@ -2,7 +2,7 @@ app_url: "https://www.vaultproject.io/"
 short_name: Vault
 long_name: Hashicorp Vault
 description: |-
-  Vault is an identity-based secrets and encryption management system. This integration Collects Vault's audit logs. The integration also collects token, memory, and storage metrics.
+  Vault is an identity-based secrets and encryption management system. This integration collects Vault's audit logs, and token, memory, and storage metrics.
```

Comment:
extra `and` can also be removed?

Reply:
if they're in one sentence they shouldn't be, since they're nested 'and's: ((audit) logs) and ((token, memory, and storage) metrics)

if you don't think it reads well, i can split back out into two sentences though!

Reply:
The explanation makes sense to me. LGTM 🚀